sorc/global_cycle.fd/land_increments.f90

```diff
@@ -549,30 +730,40 @@ subroutine apply_land_da_adjustments_soil(lsm, isot, ivegsrc,lensfc, &
 
         if (upd_stc) then
 
-          print *, 'Reverting frozen noah-mp model stc back to background'
-          n_revert=0
+          call set_soilveg_noahmp(isot, ivegsrc, maxsmc, bb, satpsi, iret)
+          if (iret < 0) then
+               print *, 'FATAL ERROR: problem in set_soilveg_noahmp'
+               call mpi_abort(mpi_comm_world, 10, ierr)
+          endif
+
+          print *, 'Adjusting noah-mp soil moisture to be consistent with soil temperature update' 
```

Comment:
I'd remove this comment. I only had the original comment to annoy us (me)  in the output so they we wouldn't forget about it.

Reply:
Removed.